infrastructure/lambda/template.yaml

```diff
@@ -140,6 +140,14 @@ Mappings:
       integration: MONTHS
       production: MONTHS
 
+  useLegacy:
```

Comment:
@ElliotMurphyGDS just realised we have added the mapping but not the parameter which uses this mapping